test/integration/production/test/index.test.js

```diff
@@ -444,38 +444,43 @@ describe('Production Usage', () => {
     it('should add prefetch tags when Link prefetch prop is used', async () => {
       const browser = await webdriver(appPort, '/prefetch')
       await waitFor(2000)
-      const elements = await browser.elementsByCss('link[rel=prefetch]')
-
-      expect(elements.length).toBe(4)
-      await Promise.all(
-        elements.map(async element => {
-          const rel = await element.getAttribute('rel')
-          const as = await element.getAttribute('as')
-          expect(rel).toBe('prefetch')
-          expect(as).toBe('script')
-        })
+
+      const elements = await browser.elementsByCss(
+        'link[rel=prefetch],link[rel=preload]'
```

Comment:
We should only search for both in Safari.

Reply:
Reverted as per https://github.com/zeit/next.js/pull/9821#discussion_r361226095